.travis.yml

```diff
@@ -182,11 +182,10 @@ jobs:
         - CCACHE_CPP2=yes
       script: echo "Not running any tests for a debug build."
 
-    # cmake build using g++-5
+    # cmake build using g++-5 + memory regression test
     - stage: Test different OS/CXX/Flags
       os: linux
       compiler: gcc
-      cache: ccache
       env:
```

Comment:
What is the reason for disabling ccache here? You changed this config to be a Debug build, but you can still use ccache with that if you increase the ccache size to 2G just for this configuration.

Reply:
Is that ok to do? Surely happy to do that, I was actually hoping for your input on that ;-)

Reply:
Yeah, I'd much rather have ccache enabled and bump it's size just for this config than not have it enabled (it'll make saving the cache back a bit slower, but you'll more than win that back with the build time improvements :-) )